aws/data_source_aws_route_test.go

```diff
@@ -50,6 +50,29 @@ func TestAccAWSRouteDataSource_TransitGatewayID(t *testing.T) {
 	})
 }
 
+func TestAccAWSRouteDataSource_LocalGatewayID(t *testing.T) {
+	var route ec2.Route
+	dataSourceName := "data.aws_route.by_local_gateway_id"
+	resourceName := "aws_route.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
```

Comment:
This will allow test environments without an available Outpost to skip the test instead of returning an error:

```suggestion
		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSOutpostsOutposts(t) },
```